zipinfo.js

```diff
@@ -55,6 +55,27 @@ ZipInfo.getEntries = function(data, dataStartOffset) {
         break;
       }
   }
+
+  // Zip64 support: if these are set, then parse as Zip64
+  if (offset === 0xFFFFFFFF || entriesLeft === 0xFFFF) {
+    if (view.getUint32(endoffset - 20, true) !== 0x07064b50) {
+      console.error('invalid zip64 EOCD locator');
+      return;
```

Comment:
The original implementation always returned an array with an offset of the root if any, while this new version does not. This different (and undocumented) return type is not ideal.